packages/editor/src/editor-ui/plugin-toolbar/plugin-toolbar.tsx

```diff
@@ -11,6 +11,7 @@ interface PluginToolbarProps {
   contentControls?: ReactElement
   pluginSettings?: ReactElement
   pluginControls?: ReactElement
+  pluginTooltip?: ReactElement
```

Comment:
Would it make more sense to just accept a string, and do the rendering of the tooltip only once in this file, instead of in each consumer?

Reply:
Thanks, great point, done